### PR TITLE
my186001 add my hometown file

### PR DESCRIPTION
**Pull request by my186001**
This is my first pull request :+1: :roller_coaster: 
Questions
- Visit Hyderabad for some fun!
- This is my first GIT HUB class.
- @crichID Thank you for your help!

### CODE REVIEW

File: my186001-hyd.md# [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# test
```
Comment (bl150003) [inline my186001-hyd.md#:1]: hi,

I found you

Binh

Comment (TDylu8) [inline my186001-hyd.md#:1]: Top 3 places to visit?

File: my186001-hyd.md# [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+##test1
+
+## My first file
+
+## Local attractions
```
Comment (my186001) [inline my186001-hyd.md#:8]: Charminar Monument,Hi Tech city

File: my186001-hyd.md# (hunk 1 above)
Comment (my186001) [inline my186001-hyd.md#:1]: Charminar Monument, Hi Tech City

**Pull request merged**